### DISCUSSION TITLE
New Black Jack game recently created!

### DISCUSSION THREAD

**Discussion by ABcodes-soon** [community/community#141798] [New to GitHub]
Hey everyone, I recently created a black jack game from the JavaScript I learned. Can you'll all give me feedback on my game?

**Comment by BrunoL0M** [comment]
ok. vere que tal tu juego

**Reply by ABcodes-soon** [reply]
How is it?

**Reply by BrunoL0M** [reply]
Well, it seems that there is still a lot of work to be done, I want to believe that in the future you will integrate the cards visually and a dealer to manage the game, I think that with those 2 types of integrations I would say that it would be a very good project
